_locales/en.xml

```diff
@@ -755,6 +755,8 @@
     <string name="pref_edit_profile">Edit Profile</string>
     <string name="disable_imap_idle">Disable IMAP IDLE</string>
     <string name="disable_imap_idle_explain">Do not use IMAP IDLE extension even if the server supports it. Enabling this option will delay message retrieval, enable it only for testing.</string>
+    <string name="enable_realtime">Enable Realtime APIs</string>
+    <string name="enable_realtime_explain"> Enable Webxdc realtime APIs which create direct connection between devices and leak IP addresses to each other.</string>
```

Comment:
changing the strings here does not work afaik, strings will be overwritten here.

the source of the strings is at https://github.com/deltachat/deltachat-android/blob/main/res/values/strings.xml , so we need a PR there.

Reply:
ftr, the strings `enable_realtime` and `enable_realtime_explain` can be pulled from transifex now